openquakeplatform/bin/oq_install.sh

```diff
@@ -0,0 +1,227 @@
+        ;;
+    ?)
+        printf "Usage: %s: [-s value] configfile\n" $(basename $0) >&2
+        exit 2
+        ;;
+    esac
+done
+shift $(($OPTIND - 1))
+
+function setup_directories() {
+    mkdir -p $GEOSERVER_DATA_DIR
+    mkdir -p $GEONODE_WWW/static
+    mkdir -p $GEONODE_WWW/uploaded
+    mkdir -p $GEONODE_WWW/wsgi
+    mkdir -p $APACHE_SITES
+    mkdir -p $GEONODE_BIN
+    mkdir -p $GEONODE_ETC
+    mkdir -p $GEONODE_ETC/media
+    mkdir -p $GEONODE_ETC/templates
+    mkdir -p $GEONODE_SHARE
+}
+
+function reorganize_configuration() {
+    cp -rp $INSTALL_DIR/support/geonode.apache $APACHE_SITES/geonode.conf
+    cp -rp $INSTALL_DIR/support/geonode.wsgi $GEONODE_WWW/wsgi/
+    cp -rp $INSTALL_DIR/support/geonode.robots $GEONODE_WWW/robots.txt
+    cp -rp $INSTALL_DIR/support/geonode.binary $GEONODE_BIN/geonode
+    cp -rp $INSTALL_DIR/GeoNode*.zip $GEONODE_SHARE
+    cp -rp $INSTALL_DIR/support/geonode.updateip $GEONODE_BIN/geonode-updateip
+    cp -rp $INSTALL_DIR/support/geonode.admin $GEONODE_SHARE/admin.json
+    cp -rp $INSTALL_DIR/support/geonode.local_settings $GEONODE_ETC/local_settings.py
+
+    chmod +x $GEONODE_BIN/geonode
+    chmod +x $GEONODE_BIN/geonode-updateip
+}
+
+function preinstall() {
+    setup_directories
+    reorganize_configuration
+    echo "Fine preinstall"
+}
+
+function randpass() {
+    [ "$2" == "0" ] && CHAR="[:alnum:]" || CHAR="[:graph:]"
+    cat /dev/urandom | tr -cd "$CHAR" | head -c ${1:-32}
+    echo
+}
+
+function setup_postgres_every_time() {
+    true # nothing to do. when we setup DB migrations they'll probably go here.
+}
+
+function setup_django_every_time() {
+    echo "setup_django_every_time:"
+    source ~/env/bin/activate
+    
+    pip -v install /usr/share/geonode/GeoNode-*.zip --no-dependencies --quiet
+    geonodedir=`python -c "import geonode;import os;print os.path.dirname(geonode.__file__)"`
+
+    ln -sf /etc/geonode/local_settings.py $HOME/env/lib/python2.7/site-packages/geonode/local_settings.py
+    ln -sf /usr/lib/python2.7/dist-packages/osgeo $HOME/env/lib/python2.7/site-packages
+
+    # Set up logging symlink
+    mkdir -p $GEONODE_LOG
+    ln -sf /var/log/apache2/error.log $GEONODE_LOG/apache.log
+
+    export DJANGO_SETTINGS_MODULE=geonode.settings
+
+    django-admin migrate account --settings=geonode.settings
+    # geonode runserver &
+    geonode migrate --verbosity 0
+    geonode loaddata $geonodedir/base/fixtures/initial_data.json
+    geonode collectstatic --noinput --verbosity 0
+    geonode createsuperuser
+
+    # Create an empty uploads dir
+    mkdir -p $GEONODE_WWW/uploaded
+    # mkdir -p $GEONODE_WWW/uploaded/thumbs/
+    ln -sf $HOME/env/lib/python2.7/site-packages/geonode/uploaded/thumbs $GEONODE_WWW/uploaded
+    mkdir -p $HOME/env/local/lib/python2.7/site-packages/geonode/uploaded
+    mkdir -p $HOME/env/local/lib/python2.7/site-packages/geonode/uploaded/layers
+    mkdir -p $HOME/env/local/lib/python2.7/site-packages/geonode/uploaded/documents
```

Comment:
All this data should not be in `site-packages`. It should contain only (almost) immutable python code and _not_ data.
What you are doing here should be managed via `collectstatic` and similar `python manage.py` commands.

I would expect this data to be under `/var/www/somewhere`